corehq/apps/data_analytics/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class MALTRow(models.Model):
+    """
+        Specifies a row for 'Monthly Aggregate Lite Table (MALT)'
+        See https://docs.google.com/document/d/1QQ3tzFPs6TWiPiah6YUBCrFILKih6OcJV7444i50o1U/edit
+    """
+    month = models.DateField()
```

Comment:
I think we should add a `db_index` to this field at a minimum, since generating these per month is a common use case